container-storage-setup.sh

```diff
@@ -34,8 +34,8 @@ STORAGE_IN_FILE="/etc/sysconfig/docker-storage-setup"
 STORAGE_OUT_FILE="/etc/sysconfig/docker-storage"
 STORAGE_DRIVERS="devicemapper overlay overlay2"
 
-PIPE1=/run/css-fifo1
-PIPE2=/run/css-fifo2
+PIPE1=/run/css-$$-fifo1
+PIPE2=/run/css-$$-fifo2
```

Comment:
So we keep on creating those pipes and never clean these up? Given we can have many more pipes now, I think it is a good idea to remove these once we are done?